javascript/big-practice/src/scripts/models/loginModel.js

```diff
@@ -0,0 +1,45 @@
+import { getUsersByUsername, getUsersById } from '../utils/fetchAPI';
+import LocalStorage from '../utils/localStorage';
+import STORAGE_KEYS from '../constants/storageKeys';
+import { EMAIL, PASSWORD } from '../constants/message';
+
+export default class LoginModel {
+  /**
+   * @description function check username and password is exists in data
+   *
+   * @param {String} email is email take from input email login
+   * @param {String} password is password take from input password login
+   *
+   * @returns {String} message
+   */
+  static async checkUser(email, password) {
+    const users = await getUsersByUsername(email);
+    let message;
+
+    if (users.length) {
```

Comment:
You should split it into 2 cases. One for check email exists or not. One for password. Shouldn't group 2 cases